rpm-build/SPECS/securedrop-workstation-dom0-config.spec

```diff
@@ -56,6 +57,7 @@ install -m 644 sd-proxy/* %{buildroot}/srv/salt/sd/sd-proxy/
 install -m 644 sd-whonix/* %{buildroot}/srv/salt/sd/sd-whonix/
 install -m 644 sd-workstation/* %{buildroot}/srv/salt/sd/sd-workstation/
 install -m 644 sys-firewall/* %{buildroot}/srv/salt/sd/sys-firewall/
+install sys-usb/* %{buildroot}/srv/salt/sd/sys-usb/
```

Comment:
Could we explicitly set the mask here for consistency?

Reply:
Will do.